src/katello/agent/katelloplugin.py

```diff
@@ -162,7 +167,7 @@ def update_settings():
     # the client certs
     ca_candidates = [os.path.join(ca_cert_dir, 'katello-default-ca.pem'),
                      os.path.join(ca_cert_dir, 'candlepin-local.pem'),
-                     rhsm_conf['rhsm']['repo_ca_cert'] % rhsm_conf['rhsm']]
+                     rhsm_conf['rhsm']['repo_ca_cert'] % {'ca_cert_dir': ca_cert_dir}]
```

Comment:
It was done mainly as that file has changed a couple times and we wanted to support older file locations, not sure if that helps.  As long as you don't see any glaring python mistakes, i'll have someone sanity test it :)